src/Tmds.ExecFunction/Tmds.ExecFunction.csproj

```diff
@@ -13,4 +13,23 @@
     <Copyright>Tom Deseyn</Copyright>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>1701;1702;NU5100</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\VsDebugger\VsDebugger.csproj">
+      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+      <SkipGetTargetFrameworkProperties>true</SkipGetTargetFrameworkProperties>
+      <Private>true</Private>
+    </ProjectReference>
+  </ItemGroup>
+
+  <ItemGroup Label="FilesToCopy">
+    <Content Include="Tmds.ExecFunction.targets" PackagePath="build/Tmds.ExecFunction.targets" />

```

Comment:
What does the `.targets` file do?

Reply:
The `.targets` file handles getting the VsDebugger into the bin folder.